### PR TITLE
grammar tweaks in docs for output.R

### PR DESCRIPTION
**Pull request by robertzk**
Let me know if some of these are undesirable, there is a whole lot here. I am not trying to slip things in like congress.

### CODE REVIEW

File: R/output.R [hunk 1]
```diff
@@ -28,12 +28,12 @@
 #' options, etc. The syntax is defined in a pattern list. All built-in pattern
 #' lists can be found in \code{all_patterns} (call it \code{apat}). First
 #' \pkg{knitr} will try to decide the pattern list based on the filename
-#' extension of the input document, e.g. \samp{Rnw} files use the list
+#' extension of the input document, e.g. \samp{Rnw} files uses the list
```
Comment (yihui) [inline R/output.R]: Is this OK?

Comment (robertzk) [inline R/output.R]: Oops! Thank you, my eyes must have glazed over the `samp` call.

**Comment by robertzk** [comment]
@yihui Addressed your comments. How does this look now?

File: R/package.R [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+#' This is an alternative tool to Sweave with a more flexible design and new
+#' features like caching and finer control of graphics. It is not limited to LaTeX
 #' and is ready to be customized to process other file formats. See the package
 #' website in the references for more information and examples.
 #' @docType package
 #' @name knitr-package
 #' @aliases knitr
 #' @author Yihui Xie <\url{http://yihui.name}>
-#' @seealso The core function in this package: \code{\link{knit}}. If you are an
+#' @seealso The core function in this package: \code{\link{knit}}. If you are a
```
Comment (yihui) [inline R/package.R]: I think it is `an` here, since we pronounce Sweave as S - weave [es wi:v] instead of [swi::v].

Comment (robertzk) [inline R/package.R]: Ah, I understand now!

**Comment by robertzk** [comment]
Final version I hope!

**Pull request merged**